### PR TITLE
docs: Ability to run the local-drone-control-java sample as multi node

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: samples/grpc/local-drone-control-java/README.md [hunk 1]
```diff
@@ -46,3 +46,24 @@ Inspect the current state of the local delivery queue
...
+instance for all nodes to use for storage and create schema for it:
+
+```shell
+docker compose up --wait
+docker exec -i local_drone_control_db psql -U postgres -t < ddl-scripts/create_tables.sql
+```
+
+Start 3 nodes, in separate terminals:
+
+FIXME choose the right main
```
Comment (pvlugter) [inline samples/grpc/local-drone-control-java/README.md:62]: Already fixed with the profile?

**Review by pvlugter — APPROVED** [review]
LGTM

**Comment by pvlugter** [comment]
Needs a simple merge with the native profile in the pom file.

File: samples/grpc/local-drone-control-java/README.md (hunk 1 above)
Comment (johanandren) [reply samples/grpc/local-drone-control-java/README.md]: FIXME: remove the FIXME

**Pull request merged**